libs/elements/layout/texts/src/lib/elewa-text-content-item/elewa-text-content-item.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="content-item">
+  <h2 class="title">{{ content.title }}</h2>
+  <p *ngFor="let description of content.descriptions; let last = last">
+    {{ description }}<ng-container *ngIf="!last"> </ng-container>
+  </p>
```

Comment:
Quick question, why do you have an ng-content here? 
Maybe add a comment explaining it. This can also be done in other code snippets that are hard to understand.